front/src/NotificationWindow.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "styled-components";
+import faTimes from "@fortawesome/fontawesome-free-solid/faTimes";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+
+const Window = styled.div`
+  background-color : ${props => props.backgroundColor}
+  border-style: solid;
+  border-size: 4px;
+  width: 90%;
+  position: relative;
+  margin-left:auto;
+  margin-right:auto;
+`;
+
+const NotificationHeader = styled.p`
+  text-align: center;
+  font-size: 5vmin;
+`;
+
+const NotificationText = styled.p`
+  padding-left: 10px;
+  padding-right: 10px;
+  text-align: center;
+  font-size: 8vmin;
+`;
+
+const NotificationCloseButton = styled.button`
+  border: none;
+  background-color: transparent;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 10vmin;
+  height: 10vmin;
+  &:focus {
+    outline: 0;
+  }
```

Comment:
we discussed this, right? 
we do not set `outline` to none. check https://github.com/rvolosatovs/turtlitto/blob/master/DEVELOPMENT.md#general

Reply:
I looked over it, will fix it this weekend.